### PR TITLE
fixed an issue with disappearing logo when resizing

### PR DESCRIPTION
**Pull request by Aykerimm**
Fixed a logo disappearing issue that occurred when window was resized.

### CODE REVIEW

File: src/styles/NavHeader.module.less [hunk 1]
```diff
@@ -26,6 +26,13 @@
...
     margin: 0 0 0 2rem;
     padding: 0;
+    @media only screen and (max-width: 1000px) {
+      margin: 0 0 0 1rem;
+      
+    }
+    @media only screen and (max-width: 900px) {
+      margin: 0 0 0 0.5rem;
+    }
   }
```
Comment (IanSaucy) [inline src/styles/NavHeader.module.less:36]: @toucham What are your thoughts on this fix?

**Comment by IanSaucy** [comment]
The disappearing logo appears to be fixed! 

I noticed that the menu bar gets cut off now though sometimes: 

image

File: src/styles/NavHeader.module.less (hunk 1 above)
Comment (toucham) [reply src/styles/NavHeader.module.less:36]: lgtm; however, `1024px` looks a bit wonky
Screen Shot 2022-06-16 at 22 50 59

**Comment by Aykerimm** [comment]
The menu looks normal when its opened the first time, and when I resize the window several times to switch from mobile to desktop view, menu seems to be moving to the right side. I have checked margins on the same width and they do not seem to work consistently. Do you have any idea why it could be behaving like that?
Changed the logo’s minimum width to 8rem, and decreased margins on menu items.

**Comment by IanSaucy** [comment]
Awesome, looks like this fixed it! I'll approve this and then you should be able to merge it via the button -- that way you get the merge commit!

**Review by IanSaucy — APPROVED** [review]
lgtm

**Review by toucham — APPROVED** [review]
lgtm

**Pull request merged**